### PR TITLE
Fix/online events

### PR DESCRIPTION
**Pull request by ramonaspence**
The online events were not coming in because of the way they were being formatted into the event dictionary on lines 223 - 237. So I just made an it an if else statement, allowing the events with no venue_id be added to the dictionary. 

The name of the days_in_the_past has been changed in the config.ini.example

### CODE REVIEW

**Comment by ramonaspence** [comment]
I started to fix the block of code that defaults the start_date of the query to today so that it defaults to `days_in_the_past`

In the process, Flask is giving me a lot of small little errors (keyerrors, typeerrors) to work out before being able to debug further. 

When I run the Flask app, I'm getting a consistent 500 response code, but I believe this only requires some refactoring of the app.

File: config.ini.example [hunk 1]
```diff
@@ -9,4 +9,4 @@ username = user
 update_cal_location = /home/user/upstate_tech_cal_service/update_cal_data.py
 
 [past_events]
-past_days = 5
\ No newline at end of file
+days_in_the_past = 5
```
Comment (allella) [inline config.ini.example:12]: Has this name been changed in app.py too where it's getting the value from the config.ini?

Comment (ramonaspence) [reply config.ini.example:12]: oops! I didn't have that in these commits. Let me add them now.

**Comment by ramonaspence** [comment]
I've also fixed a couple of small TypeErrors that Flask did not like. The app now runs on a local server with status 200.

**Comment by allella** [comment]
Cool. Squashed and merged this into this "upstream" master branch.  That's the branch the production website is "tracking" so when I do a `get pull` on the master branch of the production site it will get this commit.

**Comment by allella** [comment]
image

**Comment by allella** [comment]
If you check out your local  _.git/config_ file you'll see the git mappings for _origin_ and any "upstreams" you added.

In this case, I added a bcsnipes "upstream" when @bcsnipes was helping out, which allowed me to checkout a copy of his fork to test something and then switch back to the local master branch when I was done.

image

**Comment by ramonaspence** [comment]
Neat! I'm starting to get the hang of it/ a good grasp on it!
I like the idea of making different/new branches for certain fixes then deleting them when the fix is complete/merged as suggested in the Chapter repo you sent me. 

Although naming things clearly is vital and somehow really difficult lol!

**Pull request merged**